content/en/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: OpenTelemetry Provider
+description: This task shows you how to configure Envoy proxies to print access logs with OpenTelemetry Provider.
+weight: 10
+keywords: [telemetry,logs]
+aliases:
+    - /docs/tasks/telemetry/access-log
+    - /docs/tasks/telemetry/logs/access-log/
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+After completing this task, you understand how to have your application participate in [Envoy's access logging](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage) with [OpenTelemetry (gRPC) Access Log](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/access_loggers/open_telemetry/v3/logs_service.proto).
+Envoy proxies send access information to [otel-collector](https://github.com/open-telemetry/opentelemetry-collector)'s standard output.
+The standard output of otel-collector's containers can then be printed by the `kubectl logs` command.
+
+{{< boilerplate before-you-begin-egress >}}
+
+{{< boilerplate start-httpbin-service >}}
+
+{{< boilerplate start-otel-collector-service >}}
+
+## Enable Envoy's access logging
+
+Istio offers a few ways to enable access logs. Use of the Telemetry API is recommended
+
+### Using Telemetry API
+
+The Telemetry API can be used to enable or disable access logs:
+
+Run the following script to edit the `MeshConfig`, update the YAML files to add `otel` provider.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n istio-system -f -
```

Comment:
looking at this again, I think we should avoid trying to apply a new ConfigMap. who knows what we may accidentally overwrite.

maybe just highlight the new lines and suggest `kubectl edit cm/istio` ?

Reply:
I'm not sure how to achieve that in a doc test.

Reply:
Can we have the doc tests do this, but have the doc itself describe a more flexible process ?

Reply:
> Can we have the doc tests do this, but have the doc itself describe a more flexible process ?

We have coded tests differently in some places. In this case the docs would talk about a more general way to make the change, and the test is just a specific implementation, which should be OK.

Reply:
> Can we have the doc tests do this, but have the doc itself describe a more flexible process ?

can we add a `otel` ExtesionProvider in dmeo profile, just like the `envoy` do? I think we that would be useful when adding e2e test.

Reply:
> can we add a `otel` ExtesionProvider in dmeo profile, just like the `envoy` do? I think we that would be useful when adding e2e test.

I'm open to that, as long as we supply an `addon` yaml for generic otel collector.

Reply:
there's something [here](https://github.com/istio/istio/tree/master/samples/open-telemetry)